packages/services/storage/src/index.ts

```diff
@@ -2995,16 +2995,322 @@ export async function createStorage(connection: string, maximumPoolSize: number)
+        WHERE
+          "document_collection_id" = ${args.documentCollectionId}
+          ${
+            cursor
+              ? sql`
+                AND (
+                  (
+                    "created_at" = ${cursor.createdAt}
+                    AND "id" < ${cursor.id}
+                  )
+                  OR "created_at" < ${cursor.createdAt}
+                )
+              `
+              : sql``
+          }
+        ORDER BY
+          "document_collection_id" ASC
+          , "created_at" DESC
+          , "id" DESC
+        LIMIT ${limit + 1}
+      `);
+
+      let items = result.map(row => {
+        const node = DocumentCollectionDocumentModel.parse(row);
+
+        return {
+          node,
+          get cursor() {
+            return encodeCreatedAtAndUUIDIdBasedCursor(node);
+          },
+        };
+      });
+
+      const hasNextPage = items.length > limit;
+
+      items = items.slice(0, limit);
+
+      return {
+        items,
+        pageInfo: {
+          hasNextPage,
+          hasPreviousPage: cursor !== null,
+          get endCursor() {
+            return items[items.length - 1]?.cursor ?? '';
+          },
+          get startCursor() {
+            return items[0]?.cursor ?? '';
+          },
+        },
+      };
+    },
+
+    async createDocumentCollectionDocument(args) {
+      const result = await pool.maybeOne(sql`
+        INSERT INTO "public"."document_collection_documents" (
+          "title"
+          , "contents"
+          , "variables"
+          , "headers"
+          , "created_by_user_id"
+          , "document_collection_id"
+        )
+        VALUES (
+          ${args.title}
+          , ${args.contents}
+          , ${args.variables}
+          , ${args.headers}
+          , ${args.createdByUserId}
+          , ${args.documentCollectionId}
+        )
+        RETURNING
+          "id"
+          , "title"
+          , "contents"
+          , "variables"
+          , "headers"
+          , "created_by_user_id" as "createdByUserId"
+          , "document_collection_id" as "documentCollectionId"
+          , to_json("created_at") as "createdAt"
+          , to_json("updated_at") as "updatedAt"
+      `);
+
+      return DocumentCollectionDocumentModel.parse(result);
+    },
+
+    async deleteDocumentCollectionDocument(args) {
+      const result = await pool.maybeOneFirst(sql`
+        DELETE
+        FROM
+          "public"."document_collection_documents"
+        WHERE
+          "id" = ${args.documentCollectionDocumentId}
+        RETURNING
+          "id"
+      `);
+
+      if (result == null) {
+        return null;
+      }
+
+      return zod.string().parse(result);
+    },
+
+    async updateDocumentCollectionDocument(args) {
+      const result = await pool.maybeOne(sql`
+        UPDATE
+          "public"."document_collection_documents"
+        SET
+          "title" = COALESCE(${args.title}, "title")
+          , "contents" = COALESCE(${args.contents}, "contents")
```

Comment:
```suggestion
          , "query" = COALESCE(${args.contents}, "query")
```
I want rename this column to `query`